style.css

```diff
@@ -52,3 +52,194 @@
   border-radius: 0;
```

Comment:
[OPTIONAL] Your project has a border and is floating. It should fit on a mobile device screen without the white space below for better user experience
<img width="702" alt="Screenshot 2023-01-16 at 12 45 27" src="https://user-images.githubusercontent.com/48546992/212649053-eb789b8d-b1e7-4964-92ae-a303a13d5260.png">
It should look like this 
<img width="370" alt="Screenshot 2023-01-16 at 12 47 23" src="https://user-images.githubusercontent.com/48546992/212649120-0ae4c1b3-ea7f-4251-9392-146eef5b601f.png">